### ISSUE TITLE
[help] workaround to change file directory in init.el

### ISSUE THREAD

**Issue by zqso** [zaeph/.emacs.d#4]
Hi Zaeph, 

I’m a new Emacs and Org and Org-roam user for 1 month. I'm currently using your lisp/zp-org-agenda.el, which is such a good resource for me to refer to.

There's a file directory "\~/org/life.org" in several places within, and I need them to be "\~/Dropbox/org/life.org".

```
(let ((buffer (get-file-buffer “~/org/life.org”)
```

```
(when-let ((file (find-buffer-visiting “~/org/life.org”))
```

Is there any workaround to put in main init.el file, so that next time you update your lisp files, I don't have to search and replace them one by one.

Really appreciate for your help.

Walter

**Comment by zaeph** [comment]
Thanks for bringing this to my attention.  I'm working on it now.

**Comment by zaeph** [comment]
Fixed in 495f92949d18e9fb4554a56c7665385a92a2295f.  Make sure to define your main file `org-agenda-files` (as the first one in the list) or `zp/org-agenda-main-file` to define it explicitly.

**Comment by zqso** [comment]
> Fixed in [495f929](https://github.com/zaeph/.emacs.d/commit/495f92949d18e9fb4554a56c7665385a92a2295f). Make sure to define your main file `org-agenda-files` (as the first one in the list) or `zp/org-agenda-main-file` to define it explicitly.

Thanks Zaeph. I will try it later on, still on the go.

Just to mention, there are also:

2 similar occurrences in lisp/org-capture-web.el:

`  (let* ((template "* Notes on [[%%?%s][%s]]")
         (target '(file+headline "~/org/life.org" "Notes"))`

`          (zp/org-capture-web-create-template '(file+olp "~/org/life.org" "Film" "List")`

1 occurrence in lisp/hydra-org-refile.el:

`(defvar zp/org-agenda-files-primary nil
  "Primary org-agenda file.")

(setq zp/org-agenda-files-primary "~/org/life.org")`

If this is something that can be modified in the same way, it'd be great to see it happen.

**Comment by zaeph** [comment]
> 2 similar occurrences in lisp/org-capture-web.el:

`org-capture-web` will be deprecated in favour of [`zp-org-protocol`](https://github.com/zaeph/.emacs.d/blob/master/lisp/zp-org-protocol.el) which utilises `org-protocol` rather than kludges which hardly worked.

> 1 occurrence in lisp/hydra-org-refile.el:

I'll make this into a `defvar`.

**Comment by zaeph** [comment]
Addressed in 415a337d75546de7cd1e37e9aefb7f1491004d5a.

**Comment by zqso** [comment]
> Fixed in [495f929](https://github.com/zaeph/.emacs.d/commit/495f92949d18e9fb4554a56c7665385a92a2295f). Make sure to define your main file `org-agenda-files` (as the first one in the list) or `zp/org-agenda-main-file` to define it explicitly.

> Addressed in [415a337](https://github.com/zaeph/.emacs.d/commit/415a337d75546de7cd1e37e9aefb7f1491004d5a).

Tested, and they work perfect so far. 

Many many thanks!
